site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -15059,7 +15376,7 @@ private void doSite_copyFromTemplate(RunData data)
 		readCreateSiteTemplateInformation(params, state);
 		
 		// create site
-		doFinish(data);
+		state.setAttribute(STATE_TEMPLATE_INDEX, "18");
```

Comment:
@davidpbauer can you provide any info on the removal of doFinish? is the idea that you are not actually finished at this point? you added a new step, so now finishing happens elsewhere?

for context: we are trying to track down a serious regression in trunk. instructor that creates a site from scratch with a roster attached will have no provider attached